CodeMaid.config

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Configuration for the CodeMaid extension. IIRC, these are all the default values.

Reply:
I'm not sure if this should be part of the repository, I use codemaid extension and have a local .editorconfig myself but wouldn't commit it, same as I wouldn't commit my local color theme preferences
I may be wrong on this, this is my first time as contributor and is is taking time to learn some contributor etiquette

Reply:
Sorry, GitHub says I marked this comment as resolved. That was unintentional, and I apologize for any frustration that may have caused.

Reply:
I get where you're coming from, and certainly settings that only affect developers at an individual level shouldn't be checked in. .editorconfig can certainly be used to help make one's code consistent, but I think it's primarily meant to declare a team's desired conventions. It's no fun to open a project and find a thousand warnings because another author has broader variance. Committing an .editorconfig sets the baseline so everyone can build the code without warnings.

CodeMaid is much farther into the gray area. On further reflection, I agree that it shouldn't be checked in for OSS projects. I'll remove it.